content/2023-08-29_pyladies-byly-jsme-na-europythonu.md

```diff
@@ -0,0 +1,54 @@
+Title: PyLadies & Pyvo - Byly jsme na EuroPythonu
+Date: 2023-08-29 18:00:00
+Author: Karolina Surma, Mia Bajic
+
+Do Prahy letos poprvé doputovala konference EuroPython.
+Mezi 17. 7. a 23. 7. 2023 jste tam mohli potkat velkou část české Python komunity.
+EuroPython nám poskytl jak možnost posílit vazby v naší komunitě, tak rozšířit sítě do mezinárodních vod.
+
+S PyLadies a s Pyvem jsme to nemohly zmeškat!
+
+![samolepky_pyladies]({static}/images/samolepky_pyladies_2023.jpg)
+
+
+## Aktivity PyLadies
+
+S nápadem, že oslovíme organizátory EuroPythonu přišla **Monika Bláhová** z brněnských PyLadies.
+Vděčíme jí rozhýbání a finální rozsah prezentace PyLadies na EuroPythonu.
+Původně byl v plánu pouze večerní sraz PyLadies a příznivců, z něhož se postupně vyklubaly další aktivity.
+Realizace dopadla na výbornou díky nasazení, jaké do příprav vložila **Mia Bajić**. Díky!
+
+![aktivity_Python_komunity_na_EuPy]({static}/images/czech_python_at_eupy.jpg)
+
+
+### Hadí workshop
+
+**17. 7. 2023**, hned na začátku konference, se konal celodenní [Hadí workshop](https://ep2023.europython.eu/pyladies-workshop)
+(zmiňován již zde na blogu [v roce 2019](https://blog.python.cz/pyladies-snake-report) a [2020](https://blog.python.cz/Hadi-workshop-v-Brne]).
+Ovšem letos to bylo poprvé **anglicky**.
+Workshop opět nachystal a bravurně přednesl **Petr Viktorin** s výpomocí party **kouček a koučů** z Prahy, Plzně a Brna.
+Někteří s námi koučovali vůbec poprvé, vítáme v komunitě :)
+
+![snake_workshop]({static}/images/snake-workshop-eupy-2023.jpg)
```

Comment:
V PR je ještě `content/images/thumbnails/snake_workshop_eupy_2023.jpg`, který je ještě navíc vzhůru nohama. Hádám, že  to není nějakej easter egg :upside_down_face:

Reply:
Diky! Nemohla jsem najit, kde se stala chyba :D